### PR TITLE
Fix up CLI for exec health check

### PR DESCRIPTION
**Pull request by davidxia**
Make CLI split exec health check on spaces to match
ExecHealthCheck's interface

### CODE REVIEW

**Comment by davidxia** [comment]
@rohansingh

File: helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java [hunk 1]
```diff
@@ -406,21 +407,27 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
...
+    if (execHealthCheck != null && !execHealthCheck.isEmpty()) {
+      numberOfHealthChecks++;
+    }
+
     if (numberOfHealthChecks > 1) {
       throw new IllegalArgumentException("Only one health check may be specified.");
     }
 
-    if (!isNullOrEmpty(execHealthCheck)) {
+    if (execHealthCheck != null && !execHealthCheck.isEmpty()) {
```
Comment (rohansingh) [inline helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java]: What's wrong with `isNullOrEmpty`?

File: helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java [hunk 2]
```diff
@@ -180,7 +181,7 @@ public JobCreateCommand(final Subparser parser) {
     healthCheckExecArg = parser.addArgument("--exec-check")
         .help("Run `docker exec` health check with the provided command. The service will not be " +
               "registered in service discovery until the command executes successfully in the " +
-              "container. E.g. --exec-check=/usr/locale/bin/myhealthcheck");
+              "container, i.e. with exit code 0. E.g. --exec-check ping google.com");
```
Comment (rohansingh) [inline helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java]: Using "i.e." and "e.g." in the same line. +:100: English points.

File: helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java (hunk 1 above)
Comment (davidxia) [inline helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java]: That's only for strings right? Now this var is a List

File: helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java (hunk 2 above)
Comment (davidxia) [inline helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java]: haha

File: helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java (hunk 1 above)
Comment (rohansingh) [inline helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java]: Right, I'm an idiot. Never mind.

**Comment by rohansingh** [comment]
:+1:

**Pull request merged**